apps/aecontract/test/aecontract_SUITE.erl

```diff
@@ -556,6 +558,20 @@ init_per_testcase(TC, Config) when TC == sophia_aens_resolve;
     %% Disable name auction
     meck:expect(aec_governance, name_claim_bid_timeout, fun(_, _) -> 0 end),
     init_per_testcase_common(TC, Config);
+
+init_per_testcase(sophia_aens_update_transaction, Config) ->
+    ProtocolVsn = aec_hard_forks:protocol_vsn(?config(protocol, Config)),
+    case ProtocolVsn >= ?IRIS_PROTOCOL_VSN of
+        true ->
+            meck:expect(aec_governance, name_claim_bid_timeout, fun(_, _) -> 0 end),
+            init_per_testcase_common(sophia_aens_update_transaction,
+                                     [{sophia_version, ?SOPHIA_LIMA_FATE},
+                                      {vm_version, ?VM_FATE_SOPHIA_1},
```

Comment:
Isn't this testing the new feature? `VM_FATE_SOPHIA_1` is already released and can't have new features?

Reply:
Fixed now, we have ?VM_FATE_SOPHIA_2.